testing/cli/test_cli_sct_image.py

```diff
@@ -85,3 +86,21 @@ def test_sct_image_display_warp_check_output_exists():
     fname_out = 'grid_3_resample_' + fname_in
     sct_image.main(argv=['-i', sct_test_path('t2', fname_in), '-display-warp'])
     assert os.path.exists(sct_test_path('t2', fname_out))
+
+
+@pytest.mark.skipif(not sys.platform.startswith("linux"), reason="temporarily testing 'stitching' on just linux")
+@pytest.mark.usefixtures("run_in_sct_testing_data_dir")
+def test_sct_image_stitch():
+    """Run the CLI script and check that the stitched file was generated."""
+    # crop images for testing stitching function
+    path_in = os.path.join('t2', 't2.nii.gz')
+    fname_roi1 = 't2_roi1.nii.gz'
+    fname_roi2 = 't2_roi2.nii.gz'
+    sct_crop_image.main(argv=['-i', path_in, '-o', fname_roi1, '-xmin', '0', '-xmax', '59',
+                              '-ymin', '0', '-ymax', '20', '-zmin', '0', '-zmax', '51'])
+    sct_crop_image.main(argv=['-i', path_in, '-o', fname_roi2, '-xmin', '0', '-xmax', '59',
+                              '-ymin', '20', '-ymax', '40', '-zmin', '0', '-zmax', '51'])
+
+    fname_out = 'stitched.nii.gz'
+    sct_image.main(argv=['-i', fname_roi1, fname_roi2, '-o', fname_out, '-stitch'])
+    assert os.path.exists(sct_test_path('t2', fname_out))
```

Comment:
Ah, I think I see the reason for the test failure. `-o` is specified as just `fname_out`, but the assertion is checking against `t2/fname_out`. This is an easy fix. :)

Reply:
Resolved! :)